api/types/events/oneof.go

```diff
@@ -660,6 +660,10 @@ func ToOneOf(in AuditEvent) (*OneOf, error) {
 		out.Event = &OneOf_DatabaseUserDeactivate{
 			DatabaseUserDeactivate: e,
 		}
+	case *SpannerRPC:
+		out.Event = &OneOf_SpannerRPC{
+			SpannerRPC: e,
+		}
```

Comment:
May want to implement `TrimToMaxSize`

https://github.com/gravitational/teleport/blob/f573c2d457eea4cf7971a7e7277f6b0edc133ba8/api/types/events/events.go#L55-L58

Reply:
resolving as we discussed in slack not doing this in this PR at least